scarlet/source.py

```diff
@@ -309,18 +303,6 @@ def _make_initial(self, center, img, bg_rms, thresh=1., symmetric=True, monotoni
         except SourceInitError:
             # keep the peak sed
             logger.INFO("Using peak SED for source at {0}".format(center_int))
-
-        # By default the A matrix is normalized to unity.
-        # For the different S matrix normalizations,
-        # we make sure that our inital values are in agreement
-        # with the correct normalization
-        if self.normalization != sc.Normalization.A:
-            if self.normalization == sc.Normalization.S:
-                norm = np.sum(morph)
-            elif self.normalization == sc.Normalization.Smax:
-                norm = np.max(morph)
-            sed *= norm
-            morph /= norm
```

Comment:
Again, this will not initialize a source using S normalization properly

Reply:
yes, it will. Check `ExtendedSource`

Reply:
OK, I see your point now. Let me think...